urlparse4/cgurl.pyx

```diff
@@ -279,7 +282,7 @@ class SplitResultNamedTuple(tuple):
 class ParsedResultNamedTuple(tuple):
     __slots__ = ()
 
-    def __new__(cls, char * url, input_scheme, decoded=False):
+    def __new__(cls, char * url, input_scheme, decoded=False, canonicalized=False):
```

Comment:
``canonicalize`` would make more sense than ``canonicalized``, as nothing is canonicalized before calling this function, as I understand